content/docs/user-guide/packaging/asset-bundler/bundle-assets-for-release.md

```diff
@@ -0,0 +1,159 @@
+---
+linkTitle: Bundling Project Assets for Release
+title: Bundling Project Assets for a Project Game Release Layout
+description: Learn how to manage your assets and package them into asset bundles in Open 3D Engine (O3DE).
+weight: 150
+---
+
+This topic is an introduction to using the **Asset Bundler** to bundle assets for a distributable *project game release layout*. Managing project assets into bundles can help reduce the project's size on disk, optimize the project's runtime, and organize the assets to accommodate easy future asset releases. This topic can help you if you have a project that you're preparing to distribute for others to run or test, and you're interested in managing the project's assets before distributing it.
+
+
+## Prerequisites
+
+Build your project with the `release` configuration. This creates a *project game release layout*, which contains the application and files that you can distribute. Learn how to [create a project game release layout for Windows](\docs\user-guide\packaging\windows-release-builds).
+
+
+## Set up Asset Bundler
```

Comment:
After reading the opening paragraph I'm expecting to learn one thing - how to bundle assets. But when I continue reading, I'm confused by the formatting, especially since I haven't been given an outline of what the steps are. First, you'll notice when looking at the doc preview that there is a horizontal divider line between the "Set up" section and the "Bundle" section. The "Known Issue" callout box visually separates the two sections even further. And adding to my confusion is the choice of words in the opening to the second section: "As an introduction to the Asset Bundler tools, this section covers how to [...]". This makes it sound like maybe we've started something new here? Am I still in the tutorial? All in all I feel like there needs to be a better sense of continuity and flow here.

A few things could help this. You could use H3 headers for each of the major steps. At that header level you don't have visible horizontal breaks between each header. I'd also suggest using the same H3 level for each of the major steps, and that you introduce the steps in the intro paragraph. Something like this:

1. Set up Asset Bundler.
2. Create the game assets bundle.
3. Create the engine assets bundle.
4. Add the bundles to the release layout.

Then I'd get rid of the intro paragraph that you currently have under "Bundle content". The first sentence doesn't really add anything of value. The second sentence about the game asset bundle can be moved to step 2 ("Create the game assets bundle"). The third sentence can be moved to step 3 ("Create the engine assets bundle"). And the instruction in that step about replacing `<engine>` is actually misplaced. It belongs in the first step.